firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -219,13 +217,17 @@ private String getFingerprintHashForPackage(Context context) {
       hash = AndroidUtilsLight.getPackageCertificateHashBytes(context, context.getPackageName());
 
       if (hash == null) {
-        Log.e(TAG, "Could not get fingerprint hash for package: " + context.getPackageName());
+        LogWrapper.getInstance()
```

Comment:
I may have mispoke earlier. Can we remove this log and the log below it on line 225? These are the headers private we send to the OnePlatform API to validate the API Key, so I'm not sure we should log them. 

Instead, let's add a log wrapper around the hash log generated in the ReleaseIdentificationUtils

Reply:
The only problem is that logwrapper is package private. I have created a new logwrapper inside internal for the time being to be able to wrap the logs generated. It made sense to create a new log wrapper since its in a new package therefore the tag will be different being "FirebaseAppDistributionInternal" instead of "FirebaseAppDistribution"